app/templates/services/edit_section.html

```diff
@@ -0,0 +1,66 @@
+{% extends "_base_page.html" %}
+{% import "macros/toolkit_forms.html" as forms %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": url_for(".dashboard"),
+        "label": current_user.supplier_name
+      },
+      {
+        "link": url_for(".list_services"),
+        "label": "Services"
+      },
+      {
+        "link": url_for(".edit_service", service_id=service_id),
+        "label": service_data["serviceName"]
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      {% if error %}
+        {% with lede = error %}
+          {% include 'toolkit/forms/validation.html' %}
+        {% endwith %}
+      {% endif %}
+
+      {% with
+        heading = section.name,
+        smaller = true
+      %}
+        {% include 'toolkit/page-heading.html' %}
+      {% endwith %}
+
+    </div>
+  </div>
+  <form method="post" action="{{ url_for(".update_section", service_id=service_id, section=section.id) }}">
+
+    {% for question in section.questions %}
+      {{ forms[question.type](question, service_data, {}) }}
+    {% endfor %}
+
+    <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
```

Comment:
Would it be worth having this in some sort of "posted form" base template? Or is this edit_section template-y enough?